packages/docusaurus-plugin-content-blog/src/authorsMap.ts

```diff
@@ -93,10 +94,12 @@ export function checkAuthorsMapPermalinkCollisions(
 function normalizeAuthor({
   authorsBaseRoutePath,
   authorKey,
+  baseUrl,
   author,
 }: {
   authorsBaseRoutePath: string;
   authorKey: string;
+  baseUrl: string;
   author: AuthorInput;
 }): Author & {key: string} {
```

Comment:
Forgot to use it in another PR, but you can use type `AuthorWithKey` instead of `Author & {key: string}`